packages/react-provider/src/CSSBaseline.tsx

```diff
@@ -0,0 +1,360 @@
+import * as React from 'react';
+import { makeStaticStyles } from '@fluentui/react-make-styles';
+
+const useGlobalStyles = makeStaticStyles(`
+  /*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
This string will stay in memory forever, that's why it's minimized in Northstar: https://github.com/microsoft/fluentui/blob/master/packages/fluentui/react-northstar/src/themes/teams/staticStyles/normalizeCSS.ts